content/_index.md

```diff
@@ -56,7 +56,7 @@ Apache Camel is standalone, and can be embedded as a library within Spring Boot,
 ## Packed with Components
 Packed with several hundred components that are used to access databases, message queues, APIs or basically anything under the sun. Helping you integrate with everything.
 
-<p><a class="button-dark" href="/components/latest">Go to Component Reference</a></p>
+<p><a class="button-dark" href="/components/latest/">Go to Component Reference</a></p>
```

Comment:
I dared to take the link fixes to https://github.com/apache/camel-website/pull/456 so that we can publish Quarkus 1.0.0 announcement.